index.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: home
+paginate: true
+alt_title: "SKIP - Student Association"
+sub_title: "SKIP - Student Association"
+introduction: | 
```

Comment:
It this required that the `description` is an entry in the `YAML` header? It becomes a bit hard to read, in a way.

Reply:
I dont think its required, it can be posted as normal html as well

Reply:
I see, does, though, the `introduction` attribute support `Markdown`. Just to make sure it's correctly displayed, we don't necessarily have to migrate it to `html`.

Reply:
I migrated to html. Take a look at the difference, see what you prefer and then we can decide what we go with.

Reply:
I think it looks better now; it might have felt a bit weird to have an `html` file with no `html` code, just `yaml` header attributes. Let's keep in like this then! :)